editor/.eslintrc.js

```diff
@@ -101,7 +101,7 @@ module.exports = {
         patterns: ['**/*.spec', '**/*.spec.*'],
       },
     ],
-    'no-shadow': 'error',
+    'no-shadow': 'off', // we need to use typescript's no-shadow rule
```

Comment:
ah!

Reply:
it was picking up all enums as being shadowed, not great.